patches/com_github_square_go_jose/fix-valid-for-symmetric-key.diff

```diff
@@ -0,0 +1,15 @@
+diff --git a/jwk.go b/jwk.go
```

Comment:
Did we also submit this patch upstream? If not, please do so and put the PR number in the filename. This will make it easier to trace this.

Reply:
I originally reported the issue in their repo in June of last year via https://github.com/square/go-jose/issues/314. I didn't submit this patch as a PR because it seemed pointless to do so given how the project appears to be even more unmaintained since I first wrote this PR. I can make the attempt if you consider it worth making, but I don't expect it likely anything will come of it.

The lack of active maintenance might point toward using a different JWT library if one exists that is even minimally maintained. (Although given our earlier discussion on this PR on choice of JWT library, I'm not confident a viable replacement exists but maybe something has changed in the last few months.)

Thoughts?

Reply:
That's fine. All I'm saying: put the PR number (314) in the name of the diff: pr-314-fix-valid-for-symmetric-key.diff.

Reply:
https://github.com/square/go-jose/issues/314 is an issue, not a pull request. I had only reported the issue, given the lack of maintenance to that project, it seemed pointless to submit a PR.